stdlib/public/Concurrency/Task.swift

```diff
@@ -652,7 +652,15 @@ public func _runGroupChildTask<T>(
   // Set up the job flags for a new task.
   var flags = Task.JobFlags()
   flags.kind = .task
-  flags.priority = priorityOverride ?? getJobFlags(currentTask).priority
+
+// The ?? expression hits a codegen bug (rdar://74693488) which sometimes
+// prevents this function from returning properly. Temporarily use if/else.
+//  flags.priority = priorityOverride ?? getJobFlags(currentTask).priority
+  if let o = priorityOverride {
+    flags.priority = o
+  } else {
+    flags.priority = getJobFlags(currentTask).priority
+  }
```

Comment:
huh whoa, that explains a lot of weirdness I saw _sometimes_, thanks Mike

Reply:
I feel you. Took me ages to track down since it almost never showed up without my changes, so I was convinced that I broke it. The Universal Debugging Method (keep adding log statements until things start making sense) yielded results in the end.

Reply:
At least I feel a bit better that not only myself, new to the compiler, am led to think I lost my sanity by some of these issues sometimes. 

> The Universal Debugging Method (keep adding log statements until things start making sense) yielded results in the end.

Yeah, best (and only) method, for better or worse 🥇